.github/workflows/go.yaml

```
@@ -85,30 +85,24 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Extract Docker metadata
+        id: metadata
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=raw,value=latest,enable=${{  github.ref == 'refs/heads/main' }}
```

Based on the example [here](https://github.com/docker/metadata-action?tab=readme-ov-file#latest-tag), I'd suggest we simplify this to match the intent, but it should be functionally the same

```suggestion
            type=raw,value=latest,enable={{is_default_branch}}
```